src/components/Assets/AssetWarrantyCard.tsx

```
@@ -31,16 +31,16 @@ export default function AssetWarrantyCard(props: { asset: AssetData }) {
   }, [isCopied]);
 
   return (
-    <div className="warranty-card relative z-10 flex h-full w-screen flex-col overflow-hidden p-6 text-white transition-all hover:scale-[1.01] hover:from-primary-600 hover:to-primary-700 md:w-full md:rounded-xl xl:w-96">
+    <div className="warranty-card relative z-10 flex h-full w-full flex-col overflow-hidden p-6 text-white transition-all hover:scale-[1.01] rounded-lg hover:from-primary-600 hover:to-primary-700 xl:w-96">
       <div className="mb-3 text-right text-lg font-bold italic">
         {asset.manufacturer}
       </div>
       <div className="flex h-full flex-col justify-between gap-2 md:flex-row xl:flex-col">
         <div className="flex h-full w-full flex-col gap-4 md:border-r xl:w-auto xl:border-r-0">
           {Object.keys(details).map((key) => (
             <div className="">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)